cms/envs/common.py

```diff
@@ -2294,3 +2294,14 @@
     # The variable represents the window within which a verification is considered to be "expiring soon."
     "EXPIRING_SOON_WINDOW": 28,
 }
+
+################# Settings for brand logos. #################
+LOGO_TRADEMARK_URL = None
+LOGO_TRADEMARK_URL_PNG = None
+LOGO_TRADEMARK_URL_SVG = None
+
+LOGO_URL = None
+LOGO_URL_PNG = None
+LOGO_URL_SVG = None
```

Comment:
Can we include only the settings that are needed in `edx-platform`. For example, I don't think you need all these LOGO setting variations in this codebase, right?

Reply:
Got it, I will review this and remove the unused settings.

Reply:
Right. We can add the settings when we need them. Otherwise, I find that the monolith codebase causes further confusion if we add things that we then never use/need.

For now, you probably need: `LOGO_URL`, `LOGO_URL_PNG` (for emails), and `FAVICON_URL`.

Reply:
Got it, totally makes sense. 

`LOGO_URL` 
`LOGO_URL_PNG` (for emails)
`FAVICON_URL`
and
`LOGO_TRADEMARK_URL` (for footer logo)